<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plugin::BaseSettingsWidget</class>
 <widget class="QWidget" name="plugin::BaseSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="font_group">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="clock_font_button">
        <property name="text">
         <string>use clock font</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="custom_font_button">
          <property name="text">
           <string>custom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="choose_font_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="zoom_group">
     <property name="title">
      <string>Zoom</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="not_zoom">
        <property name="text">
         <string>do not zoom</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="font_autosize">
        <property name="text">
         <string>calculate font size</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="clock_zoom">
        <property name="text">
         <string>use clock zoom</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>custom_font_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>choose_font_button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
